daemon/src/projection.rs

```diff
@@ -367,8 +374,9 @@ impl Cfd {
             CommitConfirmed => {
                 // pretty weird if this is not defined ...
                 if let Some(dlc) = self.latest_dlc.as_ref() {
-                    self.details.tx_url_list.push(TxUrl::new(
-                        dlc.commit.0.txid(),
+                    self.details.tx_url_list.push(TxUrl::from_transaction(
```

Comment:
📣 Same as with the lock transaction, the commit transaction has a single output which does not belong to either party and corresponds to a shared output, so this won't highlight anything.

Reply:
I've added a debug_assert to prevent anyone from trying using this with commit transaction (along with explanation)